torch_geometric/nn/models/basic_gnn.py

```diff
@@ -182,6 +184,9 @@ def inference(self, loader: NeighborLoader,
         assert len(loader.num_neighbors) == 1
         assert not self.training
         # assert not loader.shuffle  # TODO (matthias) does not work :(
+        if progress_bar:
+            pbar = tqdm(total=len(self.convs))
```

Comment:
```suggestion
            pbar = tqdm(total=len(self.convs) * len(loader))
```
Let's update based on inner and outer loop.